src/extractors/collect-all-pages.js

```diff
@@ -30,7 +30,6 @@ export default async function collectAllPages({
       html,
       $,
       metaCache,
-      contentOnly: true,
```

Comment:
What is the logic behind removing this value?

Reply:
Sorry, I do not recall at this point. Obviously if I thought removing that value a good change I should have included comments around it. But I did not, and I did this over a year ago now.

Reply:
FYI, it looks like there's some context for this change in #553